### ISSUE TITLE
Replacer parameter cannot be an array

### ISSUE THREAD

**Issue by Loghorn** [WebReflection/circular-json#36]
Trying to call `CircularJSON.stringify({a: 'b'}, ['a']);` results in 

````
TypeError: replacer.call is not a function
    at Object.<anonymous> (T:\test\node_modules\circular-json\build\circular-json.node.js:62:36)
    at JSON.stringify (<anonymous>)
    at Object.stringifyRecursion [as stringify] (T:\test\node_modules\circular-json\build\circular-json.node.js:178:15)
    at repl:1:14
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:440:10)
    at emitOne (events.js:120:20)

**Comment by WebReflection** [comment]
Fixed in `0.4.0`, thank you.

**Comment by Loghorn** [comment]
Thank you!
